src/Raven.Client/Documents/Operations/Identities/SeedIdentityForOperation.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Net.Http;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Http;
+using Raven.Client.Util;
+using Sparrow.Json;
+
+namespace Raven.Client.Documents.Operations.Identities
+{
+    public class SeedIdentityForOperation : IMaintenanceOperation<long>
+    {
+        private readonly string _identityName;
+        private readonly long _identityValue;
+        private readonly bool _forceUpdate;
+
+        public SeedIdentityForOperation(string identityName, long identityValue)
```

Comment:
Rename to `name` and `value` and check if name is not null or whitespace

Reply:
Can merge those two ctors? By default force update is false?

Reply:
done